react-carousel-component/dist/styles.css

```diff
@@ -0,0 +1,43 @@
+i {
+  margin: 0.2rem;
+}
+
+img {
+  object-fit: contain;
+  width: 12rem;
+  height: 12rem;
+  margin-bottom: 1.5rem;
+}
+
+.container {
+  width: 30rem;
+  height: 20rem;
+  justify-content: center;
+  align-items: center;
+  border: 2px solid black;
+}
+
+.flex {
+  display: flex;
+}
+
+.angle {
+  font-size: 2rem;
+}
+
+.image-container {
+  width: 20rem;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.fa-angle-right,
+.fa-angle-left {
+  font-size: 3rem;
+}
+
+.fa-angle-right:active,
+.fa-angle-left:active {
```

Comment:
# 💀 
avoid augmenting third-party classes. it confuses teammates because the documentation isn't the source of truth for how they work. instead, do one of the following:

- create a whole new class of your own and use that alongside the third-party class
- create a separate stylesheet named `overrides.css` and put your augmentations in there